wards.sh

```diff
@@ -5,6 +5,10 @@ set -e
 
 [[ "$1" ]] || { echo "Please specify the ChainLog Key in ASCII to inspect"; exit 1; }
 
+### Override maxFeePerGas to avoid spikes
+[[ $ETH_GAS_PRICE && $ETH_GAS_PRICE -lt $(seth basefee) ]]
+export ETH_GAS_PRICE=$((ETH_GAS_PRICE + $(seth --to-wei 200 gwei)))
```

Comment:
I would say don't `export` this, just set it like a normal variable so it's scoped to and doesn't escape the script. 

You might also consider setting the ETH_GAS_PRICE just double or triple the `seth basefee` here so that it works in high gas environments.

Reply:
done in ccbd004

Reply:
Actually you need to export, otherwise it won't be used in the consequent `seth` calls.
It won't escape the script.
The alternative is you pass it on every `seth` call like: `ETH_GAS=$ETH_GAS seth send...`

Reply:
Just to clarify the `export` won't do it one upper level of the script but it will make it a global variable for the whole script.